pyproject.toml

```diff
@@ -23,16 +23,16 @@ dependencies = [
   "certifi >= 14.5.14",
   "cffi",
   "click >=8.0, <=8.1.3",
-  "cryptography >=39.0.1,<39.1",
+  "cryptography >=41,<44",
```

Comment:
Can we require 41.0.4 though because of https://github.com/Nitrokey/pynitrokey/security/dependabot/6?

Reply:
sure! totally missed that